bug building a dep with a dep

To fail:
```
zig add wz
zig fetch
```
add the usual incantation to your `build.zig`:
```
    inline for (std.meta.fields(@TypeOf(pkgs))) |field| {
        exe.addPackage(@field(pkgs, field.name));
    }
```
then
`zig build`
and you get
```
$ zig build
./build.zig:21:24: error: expected type '[]const std.build.Pkg', found '.@build.deps.struct:5:26'
        exe.addPackage(@field(pkgs, field.name));
                       ^
./deps.zig:5:26: note: .@build.deps.struct:5:26 declared here
        .dependencies = .{
```

This can be fixed by changing the `deps.zig` to explicitly type `.dependencies` by doing something like:
```
const Pkg = @import("std").build.Pkg;
pub const pkgs = .{
  ...
           .dependencies = &[_]Pkg{
  ...
```
...all credit to @mattnite  for chatting on me with discord and leading me to how to manually fix it.

--- a/src/commands.zig
+++ b/src/commands.zig
@@ -206,7 +206,7 @@ fn recusivePrint(
     try stream.print(".path = \"{}\",\n", .{path});
     if (edge.node.dependencies.items.len > 0) {
         try indent(stream, depth + 1);
-        try stream.print(".dependencies = .{{\n", .{});
+        try stream.print(".dependencies = &[_]Pkg{{\n", .{});
 
         for (edge.node.dependencies.items) |*dep| {
             try recusivePrint(allocator, stream, dep, depth + 2);
@@ -245,6 +245,7 @@ pub fn fetch(cache_path: ?[]const u8) !void {
 
     const file_stream = gen_file.outStream();
     try file_stream.writeAll(
+        \\const Pkg = @import("std").build.Pkg;
         \\pub const pkgs = .{
         \\
     );

